cfme/control/explorer/policies.py

```diff
@@ -376,21 +376,21 @@ def assign_actions_to_event(self, event, actions):
                 TRUE section. If :py:class:`dict`, the action is key and value specifies its
                 placement. If it's True, then it will be put in the TRUE section and so on.
         """
-        true, false = [], []
```

Comment:
Actually these "true, false"  were not invented by me https://github.com/ManageIQ/integration_tests/blame/7c2dce82524be9598e45e42f820447d66febd681/cfme/control/explorer.py#L752, but I admit blind copy/paste is bad.